[Container Registry] Test fixes (#17847)
Azure/azure-sdk-for-python: sdk/containerregistry/azure-containerregistry/samples/README.md

urlFragment: containerregistry-samples
---

> These code samples show common scenario operations with the Azure Container Registry client library. The code samples assume an environment variable `CONTAINERREGISTRY_ENDPOINT` set including the name of the login server and the `https://` prefix. For more information on using AAD with Azure Container Registry, please see the service's [Authentication Overview](https://docs.microsoft.com/azure/container-registry/container-registry-authentication).

Reviewer: Not sure if this is just me or not but I think this sentence should be like:
```
The code samples assume an environment variable `CONTAINERREGISTRY_ENDPOINT` is set, which includes
```

Author: Updated my poor grammar, thanks Catalina!

> These code samples show common scenario operations with the Azure Container Registry client library. The code samples assume an environment variable `CONTAINERREGISTRY_ENDPOINT` set including the name of the login server and the `https://` prefix. For more information on using AAD with Azure Container Registry, please see the service's [Authentication Overview](https://docs.microsoft.com/azure/container-registry/container-registry-authentication).

Author: ```suggestion
The code samples assume an environment variable `CONTAINERREGISTRY_ENDPOINT` is set, which includes the name of the login server and the `https://` prefix. For more information on using AAD with Azure Container Registry, please see the service's [Authentication Overview](https://docs.microsoft.com/azure/container-registry/container-registry-authentication).
```
